docs/rules/no-useless-computed-key.md

```diff
@@ -43,6 +41,35 @@ var c = { a: 0 };
 var c = { '0+1,234': 0 };
 ```
 
+## Options
+
+This rule has an object option:
+
+* `enforceForClassMembers` set to `true` additionally applies this rule to class members (Default `false`).
+
+### enforceForClassMembers
+
+By default, this rule does not check class declarations and class expressions,
+as the default value for `enforceForClassMembers` is `false`.
+
+When `enforceForClassMembers` is set to `true`, the rule will also disallow unnecessary computed
+keys inside of class methods, getters and setters.
+
+Examples of **incorrect** code for `{ "enforceForClassMembers": true }`:
+
+```js
+/*eslint no-useless-computed-key: ["error", { "enforceForClassMembers": true }]*/
+
+class Foo {
+    [0]() {}
+    ['a']() {}
+    get ['b']() {}
+    set ['c']() {}
```

Comment:
Can we please add a param to the setter, to avoid a syntax error in the example.